### DISCUSSION TITLE
Is fig-alt implemented for graphviz diagrams?

### DISCUSSION THREAD

**Discussion by felixvuo** [quarto-dev/quarto-cli#9000] [Q&A] [answered]
### Description

Here's a minimal document with a graphviz dot diagram.  The `fig-cap` seems to work ok, but `fig-alt` does not give alternative text in the docx or html (and obviously not in pdf).

Is this something that should work?  It would be really helpful - part of the motivation behind using quarto is to be able to produce accessible alternative formats for lectures, talks, etc.

````
---
title: "Minimum Working Example - Alt text"
author: "Me Myself"
format: 
  html:
    minimal: true
  beamer: 
    aspectratio: 169
  docx:
    toc: true
---

# First Things First

Here's a lovely diagram of something.

```{dot}
//| label: fig-tree
//| fig-alt: "A search tree rooted on node G"
//| fig-cap: "A game search tree"

graph G {
    node [label=""];    
    G -- a1;
    G -- a2;
    a1 -- a11;
    a1 -- a12;
    a1 -- a13;
    a2 -- a21;
    a2 -- a22;   
    G [label="G"];
}
```

That's all.
````

**Comment by cderv** [comment]
This is not yet supported. I'll make your request an issue so that we track this. 

Thanks !

**Reply by mcanouil** [reply, marked as answer]
Closing in favour of

- https://github.com/quarto-dev/quarto-cli/issues/9003
